packages/concerto-core/lib/introspect/modelfile.js

```diff
@@ -746,6 +746,7 @@ class ModelFile extends Decorated {
                 if (this.getModelManager().isStrict()){
                     throw new Error('Wilcard Imports are not permitted in strict mode.');
                 }
+                // Should we change this as well?
```

Comment:
Yes, let's move this to a proper deprecation too.

Reply:
Done